builtin/rebase.c

```diff
@@ -240,6 +253,7 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 		options.upstream = peel_committish(options.upstream_name);
 		if (!options.upstream)
 			die(_("invalid upstream '%s'"), options.upstream_name);
+			options.upstream_arg = options.upstream_name;
```

Comment:
The indentation of this statement is misleading... It is *not* under the `if (!options.upstream)` condition.